lib/tasks/add_navigation_links.rake

```diff
@@ -1,25 +1,69 @@
 # rubocop:disable Metrics/BlockLength
 namespace :navigation_links do
+  reading_icon = File.read(Rails.root.join("app/assets/images/twemoji/drawer.svg")).freeze
```

Comment:
I moved the link icons further up, to avoid repeating them in my task.

Reply:
It's probably just code golf, but you could make a local helper method for `image_content(*paths)` and save a few bytes - the `image_content` exists for the one rainbowdev image not in the twemoji directory.

```ruby
def image_content(*paths)
  File.read(Rails.root.join("app/assets/images/#{paths.join('/')}")).freeze
end

def twemoji(name)
  image_content("twemoji", "#{name}.svg")
end

reading_icon = twemoji("drawer")
contact_icon = twemoji("contact")
...
```

I'm sure this achieves the "unlikely to have a typo on only one line" goal of removing duplication while providing little concrete value in added readability.

Reply:
I actually quite like this suggestion @djuber.